docs/deploy-with-olm.md

```diff
@@ -109,6 +109,37 @@ the Block CR, you may run in to issues where the pod reports being unschedulable
 
 In this example, the disks were prevsiously used, and a filesystem was applied.  Be sure to either use fresh disks, or remove any partitions and file systems if you're setting up a Block mode CR.
 
+### Create a CR using Tolerations
+
+The following example assumes that some nodes in the cluster have [taints](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) that designate them for use with the local-storage Operator.
```

Comment:
Should we point to OKD docs page instead?

Reply:
Updated to use the `latest` version of the OpenShift docs.